src/standalone/Soil/Biogeochemistry/co2_parameterizations.jl

```diff
@@ -82,7 +82,7 @@ function co2_diffusivity(
     T_ref = FT(LP.T_0(earth_param_set))
     P_ref = FT(LP.P_ref(earth_param_set))
     θ_a = volumetric_air_content(θ_w, ν)
-    D0 = D_ref * (T_soil / T_ref)^FT(1.75) * (P_ref / P_sfc)
+    D0 = D_ref * max((T_soil / T_ref), 0)^FT(1.75) * (P_ref / P_sfc)
```

Comment:
This one is a true error, often caused by instability. On GPU this would give NaNs, on CPU it will break...

maybe here is another place where we should break (or return NaN?) and give a good error message, not keep going?